installer/scripts/service_control

```diff
@@ -59,6 +108,33 @@ setup_variables()
     OMSADMIN_CONF=$CONF_DIR/omsadmin.conf
 
     OMSAGENT_WS=omsagent-$WORKSPACE_ID
+
+    WS_STATUS=0
+}
+
+validate_omsadmin_conf() {
+    local _fSpec=$1
+    local _boolOnly=$2
+    VALIDATION_SUPPLEMENT_MESSAGE="Onboarding incorrect or incomplete."
+    if [ -f $_fSpec ]; then
+        validate_file_has_line_pattern $_fSpec "WORKSPACE_ID=$REGEX_UUID" $_boolOnly
+        validate_file_has_line_pattern $_fSpec "AGENT_GUID=$REGEX_UUID" $_boolOnly
+        validate_file_has_line_pattern $_fSpec "LOG_FACILITY=.*" $_boolOnly
+        validate_file_has_line_pattern $_fSpec "CERTIFICATE_UPDATE_ENDPOINT=.*" $_boolOnly
+        validate_file_has_line_pattern $_fSpec "URL_TLD=.*" $_boolOnly
+        validate_file_has_line_pattern $_fSpec "DSC_ENDPOINT=.*" $_boolOnly
+        validate_file_has_line_pattern $_fSpec "OMS_ENDPOINT=.*" $_boolOnly
+        validate_file_has_line_pattern $_fSpec "AZURE_RESOURCE_ID=.*" $_boolOnly
+        validate_file_has_line_pattern $_fSpec "OMSCLOUD_ID=.*" $_boolOnly
+        validate_file_has_line_pattern $_fSpec "UUID=$REGEX_UUID" $_boolOnly
+        return 0
+    else
+        if [ -z "$_boolOnly" ]; then
+            echo "WARNING:  OMS Admin configuration file $OMSADMIN_CONF NOT Found!" >&2
+            echo $VALIDATION_SUPPLEMENT_MESSAGE
+        fi
+        exit 2
```

Comment:
Same comment about defining what exit codes mean earlier in file

Reply:
ok